<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>current </string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>561</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>561</width>
          <height>36</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>                          Welcome </string>
        </property>
       </widget>
       <widget class="QPushButton" name="b2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>460</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>disconnect</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>721</x>
          <y>464</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_T">
        <property name="geometry">
         <rect>
          <x>153</x>
          <y>236</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>236</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>temperature</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>295</x>
          <y>236</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Luminosit√©
</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>295</x>
          <y>349</y>
          <width>48</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>pression</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_H">
        <property name="geometry">
         <rect>
          <x>153</x>
          <y>349</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>349</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>humidite</string>
        </property>
       </widget>
       <widget class="QPushButton" name="b1">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>460</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
       <widget class="QPushButton" name="b4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>460</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>delet</string>
        </property>
       </widget>
       <widget class="QPushButton" name="b3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>460</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>load data</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_lum">
        <property name="geometry">
         <rect>
          <x>437</x>
          <y>236</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_pres">
        <property name="geometry">
         <rect>
          <x>437</x>
          <y>349</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="history">
       <attribute name="title">
        <string>history</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true"/>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
